### PR TITLE
test(*): run unit tests on github

### PR DESCRIPTION
**Pull request by MingboPeng**

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ jobs:
       - name: Run Unit Tests
         working-directory: src/PollinationSDK.Tests
         run: |
+          $Env:PollinationApiKey = ${{secrets.POLLINATION_STAGING_APIKEY}}
```
Comment (MingboPeng) [inline .github/workflows/cd.yaml:27]: Hi @AntoineDao, currently I generate this ApiKey from my account on staging server. It seems working well for running unit tests. But it expires very quick. Can we have an option for setting up the expire date or have a developer's ApiKey that doesn't expire (maybe expire every 12 months)?

**Comment by AntoineDao** [comment]
Oh... This is gonna be awkward but I really don't think we should be running unit tests against a live server... I would call those integration tests and they are likely to be very brittle/unreliable when we change the server in the future.

@mostaphaRoudsari and @tymokvo do you guys have any thoughts on this?

File: .github/workflows/cd.yaml (hunk 1 above)
Comment (AntoineDao) [reply .github/workflows/cd.yaml:27]: The API key never expires so I am not sure what is going wrong here 😅

**Comment by tymokvo** [comment]
Agreed, test suites should mock external functionality.

**Pull request merged**